src/vanilla/ClientModelExtensions.cs

```diff
@@ -713,7 +713,7 @@ public static string ConstructMapper(this IModelType type, string serializedName
                     var constraintValue = constraints[keys[j]];
                     if (keys[j] == Constraint.Pattern)
                     {
-                        constraintValue = $"'{constraintValue}'";
+                        constraintValue = $"'{constraintValue.EscapeSingleQuotes()}'";
```

Comment:
@andrew-buckley - This fix should resolve the issue that you saw with the Devices swagger.